CHANGELOG.md

```diff
@@ -331,6 +331,9 @@ Read the [SwiftGen 6.0 Migration Guide](Documentation/MigrationGuide.md#swiftgen
   [SwiftGen/templates#84](https://github.com/SwiftGen/templates/pull/84)
   [Txai Wieser](https://github.com/txaiwieser)
   [SwiftGen/templates#81](https://github.com/SwiftGen/templates/pull/81)
+* Templates to generate ObjC source for .strings
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/0fbaf26f1396f63c220faa7e4722930687024d40/CHANGELOG.md#L334"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>